### PR TITLE
Fix geocoder zoom to search location.

### PR DESCRIPTION
**Pull request by kailamjeter**
Listen on lat/long change in the map and zoom to new coordinates any change in lat/long.

Screencast: https://watch.screencastify.com/v/vTZFxuZqJueO1EMrW2nY

### CODE REVIEW

File: client/src/components/NationwideMap.vue [hunk 1]
```diff
@@ -70,6 +77,15 @@ export default defineComponent({
     },
   },
   methods: {
+    zoomToLongLat() {
+      if (this.geoState?.geoids?.lat != null && this.geoState?.geoids.long != null) {
+        const longLat: LngLatLike = {
+          lon: parseInt(this.geoState?.geoids?.long),
```
Comment (justinebreuch) [inline client/src/components/NationwideMap.vue:83]: super nit: "long" so it's consistent

File: client/src/components/NationwideMap.vue [hunk 2]
```diff
@@ -285,6 +301,13 @@ export default defineComponent({
     routerLayer: function(newLayer: string) {
       this.setDataLayerVisibility(newLayer, true);
     },
+    // Listen for changes to lat/long to update map location.
+    'geoState.geoids.lat': function() {
+      this.zoomToLongLat();
+    },
+    'geoState.geoids.long': function() {
```
Comment (justinebreuch) [inline client/src/components/NationwideMap.vue:308]: You cracked the code for nested listening!!

**Review by justinebreuch — APPROVED** [review]
Thanks for doing this!

File: client/src/components/NationwideMap.vue (hunk 1 above)
Comment (kailamjeter) [reply client/src/components/NationwideMap.vue:83]: It actually has to be lon for mapbox type inference, but I changed other references to lon for consistency.

File: client/src/components/NationwideMap.vue (hunk 2 above)
Comment (kailamjeter) [reply client/src/components/NationwideMap.vue:308]: another vue quirk!

**Pull request merged**